iotdb-core/node-commons/src/test/java/org/apache/iotdb/commons/utils/NodeUrlUtilsTest.java

```diff
@@ -59,4 +59,37 @@ public void parseAndConvertTConfigNodeUrlsTest() throws BadNodeUrlException {
     Assert.assertEquals(configNodeUrls, NodeUrlUtils.convertTConfigNodeUrls(configNodeLocations));
     Assert.assertEquals(configNodeLocations, NodeUrlUtils.parseTConfigNodeUrls(configNodeUrls));
   }
+
+  @Test
+  public void parseAndConvertTEndPointUrlsIPV4AndIPV6Test() throws BadNodeUrlException {
+    final List<TEndPoint> endPoints =
+        Arrays.asList(
+            new TEndPoint("AD80:E32B:CR25:B3WE:DG4G:DWTF:CGDE:2345", 6667),
+            new TEndPoint("0:0:0:0:0:FFFF:129.144.52.38", 6668),
+            new TEndPoint("::13.1.68.3", 6669));
+    final String endPointUrls =
+        "AD80:E32B:CR25:B3WE:DG4G:DWTF:CGDE:2345:6667,[0:0:0:0:0:FFFF:129.144.52.38]:6668,[::13.1.68.3]:6669";
```

Comment:
This can still be A-F~ and the same for the other functions in the very file.

Reply:
done